src/NServiceBus.Transport.Bridge/RawEndpoints/IErrorHandlingPolicyContext.cs

```diff
@@ -12,7 +12,7 @@ interface IErrorHandlingPolicyContext
         /// <summary>
         /// Moves a given message to the error queue.
         /// </summary>
-        Task<ErrorHandleResult> MoveToErrorQueue(string errorQueue, bool attachStandardFailureHeaders = true, CancellationToken cancellationToken = default);
+        Task<ErrorHandleResult> MoveToErrorQueue(string errorQueue, CancellationToken cancellationToken = default);
```

Comment:
I didn't analyze the code too much, so how do customers now get the exception as to why the message could not be shoveled? Is that still added somehow?

Reply:
@dvdstelt `MessageShovelErrorHandlingPolicy` logs error after 3 immediate attempts.